blueprints/modular/projects.yaml

```diff
@@ -0,0 +1,42 @@
+title: Projects
+'@extends': default
+
+form:
+  fields:
+    tabs:
+      fields:
+        features:
+          type: tab
+          title: Projects
+          fields:
+            header.projects:
+              name: projects
+              type: list
+              label: Projects
+              fields:
+                .title:
+                  type: text
+                  label: Title
+                  validate:

```

Comment:
Don't need this.

Reply:
We can delete this `validate` attribute on text fields. There are several other instances of this that I'll go ahead and mark.